testing/vcs/test_vcs_init_open_sizing.py

```diff
@@ -46,12 +46,6 @@ def test_canvas_size(c, size, via):
     info = c.canvasinfo()
     w, h = size
 
-    # Make sure size fits on screen bounds
-    screen_w, screen_h = c.backend.renWin.GetScreenSize()
```

Comment:
I think GetScreenSize either works or not in VTK... Never could figure out when it does or not, so it probably a combination of this + small screen